### PR TITLE
Stack command

### PR DESCRIPTION
**Pull request by aminusfu**

### CODE REVIEW

File: pmxbot/stack.py [hunk 1]
```diff
@@ -0,0 +1,707 @@
...
+
+    def get_items(self, topic):
+        rows = self.db.execute("SELECT items FROM stack WHERE topic = ?", [topic])
+        if not rows:
+            return []
+        else:
+            return rows[0][0].split("\n")
+
+    def save_items(self, topic, items):
+        items = "\n".join(items)
```
Comment (dhandy2013) [inline pmxbot/stack.py]: There would be extra items created if the user does something like press Ctrl-Enter to insert a newline into the stack item string. That's probably something we can live with. :)

**Comment by dhandy2013** [comment]
It seems like it would be easy to create topic that is not someone's nick, and then lose track of it. Maybe a command to list topics?

**Comment by aminusfu** [comment]
Good idea. I'll add that. I wonder how mind-blowing it would be if "topics" were itself a stack...

**Comment by aminusfu** [comment]
Merged into the fork parent.

**Pull request closed without merging**